app/misc/authenticate.ts

```diff
@@ -139,7 +139,7 @@ function getUserInfo(callback) {
           $("#otpModal").modal('show');
         });
       }
-      else if (err == "Error: getaddrinfo ENOTFOUND api.github.com api.github.com:443"){
+      else if (err.errno == "ENOTFOUND" || err.errno =="ENOENT"){
```

Comment:
Random Question: Is there a precedent for constant strings?

Reply:
The precedent in the project is this.